pkgs/games/anki/default.nix

```diff
@@ -118,35 +119,37 @@ let
   };
 in
 python3.pkgs.buildPythonApplication {
-  inherit pname version src;
+  inherit pname version;
+
+  outputs = [ "doc" "man" "out" ];
 
-  outputs = [ "out" "doc" "man" ];
```

Comment:
Why did the order of outputs change here? By not having "out" first, it means that `nix-build -A anki` links the `doc` output as `result` rather than the `out` one